Reworks takehome for full-stack (#49)
workpathco/support: dev-candidates.md

# Workpath Developer Candidate Code Project

In order to help us get a better sense of your approach to coding and ability as a developer, we have designed a small code exercise.  The goal of the exercise is to build a simple user interface for the publicly available NY Times "Top News" API.  Users should be able to do keyword searches for articles, and to select the section of the newspaper to display articles from.  

> Your submission should be a full-stack solution in a language of your preference.  You may include as many supporting files and assets as you like, but the finished project should be contained in a single zip archive or tarball.  If you decide to use a framework that produces a compiled version of your code (such as React), be sure to include your original source files with your submission.

Reviewer: ```suggestion
Your submission should be a full-stack solution in JavaScript, Ruby or Elixir.  You may include as many supporting files and assets as you like, but the finished project should be contained in a single zip archive or tarball.  If you decide to use a framework that produces a compiled version of your code, be sure to include your original source files with your submission.
```

I don't like limiting it, but I feel like we should have some knowledge of the language used so that we can have an educated discussion. 😅

Feel free to ignore if you don't agree.

Author: @epogue I agree, but what languages are we going to allow?

Author: @epogue could allow Ruby since you know it well.

Reviewer: Yep, I suggested JavaScript, Ruby, and Elixir.

Reviewer: Including JS and Ruby should make it pretty approachable. Adding Python would probably cover 95%, but I don't know Python that well right now.

I refuse to accept Java. 😅

Author: Ok yeah, that works for me too.

What's Allowed
--------------

> - You may use any frameworks, libraries or third-party modules you wish.  Your choice to use or not use third party tools is not going to impact your evaluation one way or the other.  Just keep in mind that we want to be able to see your code, not your familiarity with the language and/or framework.

Reviewer: ```suggestion
- You may use any frameworks, libraries or third-party modules you wish.  Your choice to use or not use third party tools is not going to impact your evaluation one way or the other.  Just keep in mind that we want to be able to see your code, not your familiarity with a specific framework.
```
